internal/pkg/cli/svc_package.go

```diff
@@ -24,91 +25,92 @@ import (
 )
 
 const (
-	appPackageAppNamePrompt = "Which application would you like to generate a CloudFormation template for?"
-	appPackageEnvNamePrompt = "Which environment would you like to create this stack for?"
+	svcPackageSvcNamePrompt = "Which service would you like to generate a CloudFormation template for?"
+	svcPackageEnvNamePrompt = "Which environment would you like to create this stack for?"
```

Comment:
Package this stack for?

Reply:
Addressed it here! https://github.com/aws/amazon-ecs-cli-v2/pull/922/commits/c5965430c72163727dfe4fd1c1239a82c612f096